### ISSUE TITLE
Adding `UXarray` to NPL environments

### ISSUE THREAD

**Issue by negin513** [NCAR/ncar-conda#8]
@philipc2 and @erogluorhan requested adding UXarray to NPL environment. 

Here is the log of changes required:

```
Looking for: ['uxarray']

r/noarch                                                      No change
r/linux-64                                                    No change
conda-forge/noarch                                  14.6MB @  20.1MB/s  0.8s
conda-forge/linux-64                                34.3MB @  36.4MB/s  0.9s

Pinned packages:
  - python 3.11.*


Transaction

  Prefix: /glade/u/apps/opt/conda/envs/npl-2024a

  Updating specs:

   - uxarray
   - ca-certificates
   - certifi
   - openssl


  Package            Version  Build         Channel           Size
────────────────────────────────────────────────────────────────────
  Install:
────────────────────────────────────────────────────────────────────

  + retrying           1.3.3  py_2          conda-forge       11kB
  + antimeridian       0.3.5  pyhd8ed1ab_0  conda-forge       19kB
  + spatialpandas     0.4.10  pyhd8ed1ab_1  conda-forge       78kB
  + uxarray        2024.04.0  pyhd8ed1ab_0  conda-forge      119kB

  Upgrade:
────────────────────────────────────────────────────────────────────

  - openssl            3.2.1  hd590300_1    conda-forge     Cached
  + openssl            3.3.0  hd590300_0    conda-forge     Cached

  Summary:

  Install: 4 packages
  Upgrade: 1 packages

  Total download: 228kB

────────────────────────────────────────────────────────────────────


```

**Comment by negin513** [comment]
This issue is resolved by #9, and as of now, UXarray should be available as part of `npl-2024a` and future NPL versions.

FYI @philipc2 and @erogluorhan

**Comment by philipc2** [comment]
> This issue is resolved by #9, and as of now, UXarray should be available as part of `npl-2024a` and future NPL versions.
> 
> FYI @philipc2 and @erogluorhan

Thank you for putting this together so quickly!

**Comment by erogluorhan** [comment]
This is great to see; thanks a lot @negin513 ! Honestly we were hoping to see UXarray going into NPL no sooner than the next bi-annual NPL release; it is fantastic to see it in npl-2024a
